ci/bamboo/install-pip-setuptools-wheel.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Shared script for installing specific versions of pip, setuptools, and wheel on *nix (Linux and OS X) builds.